src/Tasks/Common/Resources/xlf/Strings.tr.xlf

```diff
@@ -2,41 +2,21 @@
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="tr" original="../Strings.resx">
     <body>
-      <trans-unit id="AspNetCoreAllNotSupported">
```

Comment:
An aside: what's with the changes to this file (i.e. the shuffling) while the others have the simple additions?

Reply:
No idea 🤷‍♂️